composer.json

```diff
@@ -37,7 +37,7 @@
     ],
     "require": {
         "php": "^8.1",
-        "aic/data-hub-foundation": "dev-laravel-8-support",
+        "aic/data-hub-foundation": "dev-main",
```

Comment:
When updating composer packages to use the data hub foundation's main branch, let’s pivot to starting to use version tags.

So instead of `"aic/data-hub-foundation": "dev-main"`, make it `"aic/data-hub-foundation": "1.0"`.

(Specifically "1.0"  in this case since the commit on tag 1.1 requires Laravel 10. Since the website hasn’t been tested on version 10 yet, it shouldn’t automatically get bumped to 1.1)

Reply:
Can/should we use version 2.0 to represent the update to Laravel 10? It's possible we may need to make further changes to `data-hub-foundation` v1.0 for our Laravel 8 projects before we get around to updating them to Laravel 10.

Reply:
Yes, good call! You can change the tags around. If you'd like a hand, I can do it, just let me know.

Reply:
I removed the `1.1` tag and added a `2.0` tag to the foundation.